lemur/common/schema.py

```diff
@@ -159,7 +160,10 @@ def decorated_function(*args, **kwargs):
             except Exception as e:
                 capture_exception()
                 current_app.logger.exception(e)
-                return dict(message=str(e)), 500
+                code = 500
```

Comment:
We may be able to avoid this change by switching the order of validate_schema and permission require decorators. I'm concerned that some users may implement schema validation that relies on a remote service in which case it would be appropriate to report as a 500 even if remote returned a 4xx